pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -1100,7 +1114,7 @@ pj_status_t pjsua_vid_channel_update(pjsua_call_media *call_med,
 	    }
 	}
 
-        if (pjsua_var.ua_cfg.cb.on_stream_precreate) {
+        if (!call->hanging_up && pjsua_var.ua_cfg.cb.on_stream_precreate) {
```

Comment:
This does not look related to the codec? If it is a fix for another issue, please put it in another PR or misc PR.